docs/tasks/debug-application-cluster/events-stackdriver.md

```diff
@@ -0,0 +1,88 @@
+---
+assignees:
+- crassirostris
+- piosz
+title: Events in Stackdriver
+---
+
+
+
+Kubernetes events are objects that provide insight into what is happening
+inside a cluster, such as what decisions were made by scheduler or why some
+pods were evicted from the node. You can read more about using events
+for debugging your application in the [Application Introspection and Debugging
+](/docs/tasks/debug-application-cluster/debug-application-introspection/)
+section.
+
+Since events are API objects, they are stored in the apiserver on master. To
+avoid filling up master's disk, a retention policy is enforced: events are
+removed one hour after the last occurrence. To provide longer history
+and aggregation capabilities, a third party solution should be installed
+to capture events.
+
+This article describes a solution that exports Kubernetes events to
+Stackdriver Logging, where they can be processed and analyzed.
+
+**Note:** it is not guaranteed that all events happening in a cluster will be
+exported to Stackdriver. One example of events that are not exported are
```

Comment:
How about:

`One possible scenario when events will not be exported is when event exporter is not running (e.g. during restart or upgrade).`